import-script/roles.json

```diff
@@ -1,9 +1,16 @@
 [
+  {
+    "name": "dp-interactives-importer",
+    "permissions": [
+      "interactives:update"
```

Comment:
Does the interactives importer need to read the current state of the interactive from the API before updating it? If so you may also need to add `interactives:read` permission here as well.

Reply:
hi, no read just update